meson.build

```diff
@@ -5,15 +5,16 @@ cpu_family = target_machine.cpu_family()
 add_project_arguments('-DNOMINMAX', language : 'cpp')
 
 dxvk_compiler = meson.get_compiler('cpp')
-dxvk_msvc=dxvk_compiler.get_id() == 'msvc'
+dxvk_msvc = dxvk_compiler.get_id() == 'msvc'
```

Comment:
can we name this `is_msvc` or something?

Reply:
Done.

Reply:
wait

Reply:
Forgot to add a few files to the commit, fixed.